README.md

```diff
@@ -1 +1,82 @@
-# Portfolio-setup-and-mobile-first
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
```

Comment:
:exclamation: :exclamation: Kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections are present in your README file.
- The only optional section is the FAQ. However, I noticed that you deleted some of the sections please make sure to include all of them. This is crucial to have a professional repository. :ok_hand:
- Please remove all the instructions targeting you. For example: "Describe how a new developer could make use of your project." 
- Lastly, your MIT license doesn't contain your details. You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add an MIT license to your project properly. :wink: